src/main/java/ratpack/pac4j/internal/RatpackWebContext.java

```diff
@@ -65,22 +79,29 @@ public RatpackWebContext(Context ctx, TypedData body, SessionData session) {
     Promise<SessionData> sessionDataPromise = ctx.get(Session.class).getData();
     if (bodyBacked) {
       return ctx.getRequest().getBody().flatMap(body ->
-        sessionDataPromise.map(sessionData -> new RatpackWebContext(ctx, body, sessionData))
+          sessionDataPromise.map(sessionData -> new RatpackWebContext(ctx, body, sessionData))
       );
     } else {
       return sessionDataPromise.map(sessionData -> new RatpackWebContext(ctx, null, sessionData));
     }
   }
 
-  @Override
```

Comment:
Weird that this was removed from the interface.  Regardless, it still is needed by the implementation of this component, and there isn't a better place for it.